### PR TITLE
Modularization

### PR DESCRIPTION
**Pull request by dmroldanch**
First version / Architecture

### CODE REVIEW

File: .idea/.gitignore
Comment (iteneum) [inline .idea/.gitignore:1]: Remove all .idea folder/files from the pr

File: app/build.gradle [hunk 1]
```diff
@@ -48,16 +48,14 @@ android {
 
 dependencies {
 
+    implementation project(":feature:login")
+    implementation project(":feature:example")
+    implementation libs.bundles.compose
+
     implementation 'androidx.core:core-ktx:1.7.0'
```
Comment (iteneum) [inline app/build.gradle:55]: Move all libs and versions to the other gradle file, so everything is complaint with the standard

File: settings.gradle [hunk 2]
```diff
@@ -11,6 +11,67 @@ dependencyResolutionManagement {
...
+            library('hilt-compiler', 'com.google.dagger','hilt-compiler').versionRef('hilt')
+
+            bundle('compose', ['compose-material3', 'compose-windows', 'compose-activity', 'compose-view-model', 'compose-constraints','compose-tooling','compose-tooling-prev','compose-navigation'])
+            bundle('accompanist', ['accompanist-permisions'])
+            bundle('dataStoreBundle', ['dataStore', 'dataStore-Core'])
+        }
+    }
+
 }
 rootProject.name = "Leaspert"
```
Comment (iteneum) [inline settings.gradle:62]: Typo, the correct is "Leasepert", with an "e" after "Leas" Could you correct it everywhere please? my mistake :P

**Review by iteneum — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (dmroldanch) [reply app/build.gradle:55]: I moved those libs

File: settings.gradle (hunk 2 above)
Comment (dmroldanch) [reply settings.gradle:62]: I changed the name

**Review by iteneum — APPROVED** [review]

**Pull request merged**